zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterResult.java

```diff
@@ -46,7 +46,8 @@
     TABLE,
     IMG,
     SVG,
-    NULL
+    NULL,
+    NETWORK
```

Comment:
Adding a new interpreter result type means adding a new [DisplaySystem](http://zeppelin.apache.org/docs/latest/displaysystem/basicdisplaysystem.html)

And every display system should accessible from interpreter's output. for example, Table DisplaySystem,

```
%spark
print(s"""%table
key\tvalue
sun\t100
moon\t20""")
```

```
%sh echo -e "%table key\tvalue\nsun\t100\nmoon\t20"
```

Any output of interpreter is serialized as a string into [InterpreterResult.msg](https://github.com/apache/zeppelin/blob/v0.6.2/zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterResult.java#L54) and if `%table` is detected, it is [deserialized in a front-end side and rendered](https://github.com/apache/zeppelin/blob/v0.6.2/zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js#L109)

Can we do the same with NETWORK?

Or another approach could be remove 'NETWORK' result type, describe node/edge information some how in TABLE format. And let loadNetworkData() parse and construct network.

What do you think?